<ui version="4.0" >
 <class>QStarDict::MainWindow</class>
 <widget class="QMainWindow" name="QStarDict::MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QStarDict</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pixmaps/pixmaps.qrc" >:/icons/qstardict.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
    <property name="horizontalSpacing" >
     <number>6</number>
    </property>
    <property name="verticalSpacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="searchClearButton" >
        <property name="toolTip" >
         <string>Clear the search box</string>
        </property>
        <property name="text" >
         <string>Clear</string>
        </property>
        <property name="icon" >
         <iconset resource="pixmaps/pixmaps.qrc" >:/icons/clear-right.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBox" />
      </item>
      <item>
       <widget class="QToolButton" name="queryButton" >
        <property name="toolTip" >
         <string>Fuzzy query</string>
        </property>
        <property name="text" >
         <string>Search</string>
        </property>
        <property name="icon" >
         <iconset resource="pixmaps/pixmaps.qrc" >:/icons/search-filter.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <widget class="DictWidget" native="1" name="translationView" />
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="wordsListDock" >
   <property name="features" >
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::DockWidgetAreas(Qt::LeftDockWidgetArea|Qt::NoDockWidgetArea|Qt::RightDockWidgetArea)</set>
   </property>
   <property name="windowTitle" >
    <string>Words list</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>9</number>
     </property>
     <property name="topMargin" >
      <number>9</number>
     </property>
     <property name="rightMargin" >
      <number>9</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QListWidget" name="wordsList" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionSaveToFile" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menu_Options" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionScan" />
    <addaction name="actionSettings" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Options" />
   <addaction name="menuHelp" />
  </widget>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="pixmaps/pixmaps.qrc" >:/icons/application-exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="icon" >
    <iconset resource="pixmaps/pixmaps.qrc" >:/icons/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configure QStarDict</string>
   </property>
  </action>
  <action name="actionScan" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Scan</string>
   </property>
  </action>
  <action name="actionSaveToFile" >
   <property name="icon" >
    <iconset resource="pixmaps/pixmaps.qrc" >:/icons/document-save-as.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save to file</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DictWidget</class>
   <extends>QWidget</extends>
   <header>dictwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchBox</tabstop>
  <tabstop>queryButton</tabstop>
  <tabstop>wordsList</tabstop>
  <tabstop>searchClearButton</tabstop>
 </tabstops>
 <resources>
  <include location="pixmaps/pixmaps.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>searchClearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>315</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>647</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchClearButton</sender>
   <signal>clicked()</signal>
   <receiver>searchBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>315</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>647</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchBox</sender>
   <signal>returnPressed()</signal>
   <receiver>queryButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>438</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>661</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
